File: src/main/java/com/core/base/corebase/controller/organization/dto/OrganizationReq.java
// Copyright 2025 Kakao Healthcare Corp. All Rights Reserved.

package com.core.base.corebase.controller.organization.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.lang.Nullable;
import org.springframework.web.multipart.MultipartFile;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class OrganizationReq {
    private String name;
    
    @Nullable
    private MultipartFile logo;
    
    @Nullable
    private String ceo;
    
    @Nullable
    private String telNumber;
    
    @Nullable
    private String address;
}

File: src/main/kotlin/com/core/base/corebase/controller/organization/dto/OrganizationReq.kt
package com.core.base.corebase.controller.organization.dto

import org.springframework.lang.Nullable
import org.springframework.web.multipart.MultipartFile

class OrganizationReq(
    val name: String,
    @Nullable
    val logo: MultipartFile?,
    @Nullable
    val ceo: String?,
    @Nullable
    val telNumber: String?,
    @Nullable
    val address: String?,
)